dj_debexpo/dj_debexpo/wsgi.py

```diff
@@ -0,0 +1,14 @@
+"""
+WSGI config for dj_debexpo project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.7/howto/deployment/wsgi/
+"""
+
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "dj_debexpo.settings")
+
+from django.core.wsgi import get_wsgi_application
+application = get_wsgi_application()
```

Comment:
For embedding the Turbogears stuff into our Django views, we could use http://pythonhosted.org/twod.wsgi/embedded-apps.html#embedding-non-django-applications instead, fwiw.

I have no particular reason to trust one of these over the other.